package.json

```diff
@@ -14,8 +15,7 @@
     "coverage": "node_modules/.bin/istanbul cover test/unit/run.js",
     "audit": "npm shrinkwrap; node node_modules/nsp/bin/nspCLI.js audit-shrinkwrap; rm npm-shrinkwrap.json;",
     "docs": "./bin/generate-docs",
-    "lint": "jshint .",
-    "validate": "npm ls"
+    "lint": "jshint ."
```

Comment:
I believe if you bring in the latest changes, you'll no longer need to remove this line!